src/CosmosDB/CosmosDB/CosmosDBAccount/NewOrUpdateAzCosmosDBAccount.cs

```diff
@@ -73,6 +73,14 @@ public class NewOrUpdateAzCosmosDBAccount : AzureCosmosDBCmdletBase
         [Parameter(Mandatory = false, HelpMessage = Constants.AsJobHelpMessage)]
         public SwitchParameter AsJob { get; set; }
 
+        [Parameter(Mandatory = false, HelpMessage = Constants.NetworkAclBypassHelpMessage)]
```

Comment:
what would be the value of NetworkAclBypass for cosmos db account if this parameter was not provided?

Reply:
Default is None

Reply:
If so, I suggest using switch parameter instead of string for "-NetworkAclBypass"

Reply:
Isnt Switch for booleans ? This is not a boolean value

Reply:
set the value to "AzureServices" when "-NetworkAclBypass" is provided, "None" if not.

If I understood correctly "-NetworkAclBypass" means two states only.

Reply:
It is an enum because we may add another value to it in the future. It is better to keep it as a string instead of SwitchParameter.